src/components/views/rooms/MessageComposerInput.js

```diff
@@ -1258,28 +1256,23 @@ export default class MessageComposerInput extends React.Component {
             return;
         }
 
-        // Select history only if we are not currently auto-completing
-        if (this.autocomplete.state.completionList.length === 0) {
```

Comment:
This conflicts now with the change I did to start editing events in the timeline with arrow up, sorry. The local editor history was removed. Adapting this to your changes should be easy though (e.g. get rid of the if and only keep the positive branch)

Reply:
No worries, it probably should've ignored the whitespaces in the first place. Updated!